src/utils/stripLocationInfo.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Returns a copied version of the node recieved without location attributes.
+ * @param  {Object}  node
+ * @return {Object}  node
+ */
+export default function stripLocationInfo(node) {
```

Comment:
But you use it inside pattern passed to `matchesAst()` which in turn uses it for comparison.